src/common/components/buttons.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ButtonHTMLAttributes, forwardRef } from 'react';
+import styles from './buttons.module.css';
+
+export const PrimaryButton = createButton( styles.primary );
+
+export const ClearButton = createButton( styles.clear );
+
+export const TextButton = createButton( styles.text );
+
+export const OutlinePrimaryButton = createButton( styles.outlinePrimary );
+
+export const OutlineNeutralButton = createButton( styles.outlineNeutral );
+
+function createButton( buttonStlyeClass: string ) {
+	return forwardRef< HTMLButtonElement, ButtonHTMLAttributes< HTMLButtonElement > >(
+		function Button( { className, children, ...props }, ref ) {
```

Comment:
**nitpick (non-blocking):** We should have a default value for the `className` so we don't get `undefined` in case we don't provide it as a prop. 

For example, in the selected feature, the class name for the `Clear` button reads as ` <button class="buttons_button__Yk0Vz buttons_text__C85wU undefined" type="button" aria-label="Clear currently selected feature">Clear</button>`

Reply:
Oh shoot, nice catch, for some reason I thought that it would default to an empty string -- I'll get that added!